src/js/ads.js

```diff
@@ -180,7 +180,7 @@ class VideoAds {
 		// If ads have failed to load, which resets the advertising support flag, play the video
 		// instead; otherwise, wait until the ads have loaded.
 		if (!this.video.opts.advertising) {
-			this.playUserVideo();
+			return this.playUserVideo();
```

Comment:
On line 190 we remove the loading spinner if it exists, that would no longer happen with this early return. Perhaps we should move that code above this section.

Reply:
Ahhhh, that will be why... will go back and refactor